docs/setup/turn/eturnal.md

```diff
@@ -0,0 +1,153 @@
+    
+    The relevant lines, with example values, are:
+
+    ```yaml
+    eturnal:
+        secret: "long-and-cryptic"     # Shared secret, CHANGE THIS.
+    ```
+
+    One way to generate the `secret` is with `pwgen`:
+
+    ```sh
+    pwgen -s 64 1
+    ```
+
+1.  If your TURN server is behind NAT, the NAT gateway must have an external,
+    publicly-reachable IP address. `eturnal` tries to autodetect the public IP address, however, it may also be configured, so `eturnal` advertises that
+    address to connecting clients:
+
+    ```yaml
+        relay_ipv4_addr: "203.0.113.4" # The server's public IPv4 address.
+    ```
+
+    If your NAT gateway is reachable over both IPv4 and IPv6, you may
+    configure `eturnal` to advertise each available address:
+
+    ```yaml
+        relay_ipv4_addr: "203.0.113.4" # The server's public IPv4 address.
+        relay_ipv6_addr: "2001:db8::4" # The server's public IPv6 address (optional).
+    ```
+
+    When advertising an external IPv6 address, ensure that the firewall and
+    network settings of the system running your TURN server are configured to
+    accept IPv6 traffic, and that the TURN server is listening on the local
+    IPv6 address that is mapped by NAT to the external IPv6 address.
+
+1.  If eturnal was started by systemd, log files are written into the
+    `/var/log/eturnal` directory by default. In order to log to the [journal](https://www.freedesktop.org/software/systemd/man/systemd-journald.service.html)
+    instead, the `log_dir` option can be set to `stdout` in the configuration file.
+
+1.  Consider your security settings. TURN lets users request a relay which will
+    connect to arbitrary IP addresses and ports. The following configuration is
+    suggested as a minimum starting point, [see also the official documentation](https://eturnal.net/documentation/#blacklist):
+
+    ```yaml
+        ## Reject TURN relaying from/to the following addresses/networks:
+        blacklist:                # This is the default blacklist.
+            - "127.0.0.0/8"         # IPv4 loopback.
+            - "::1"                 # IPv6 loopback.
+            #- recommended          # Expands to a number of networks recommended to be
+                                    # blocked, but includes private networks. Those
+                                    # would have to be 'whitelist'ed if eturnal serves
+                                    # local clients/peers within such networks.
+    ```
+
+    To block further recommend address ranges, uncomment the `- recommended` part in the configuration file.
+
+1.  Also consider supporting TLS/DTLS. To do this, adjust the following settings
+    in the `eturnal.yml` configuration file (TLS parts should not be commented anymore):
+
+    ```yaml
+        listen:
+            -
+            ip: "::"
+            port: 3478
+            transport: udp
+            -
+            ip: "::"
+            port: 3478
+            transport: tcp
```

Comment:
this feels like weird YAML formatting. Does it work like that?

Call me old fashioned but I would have expected e.g.
```suggestion
            - ip: "::"
              port: 3478
              transport: udp
            - ip: "::"
              port: 3478
              transport: tcp
```

Reply:
I have it running in with initial format, however, I am fine with the changes. ;)